### PR TITLE
Add Simplified Chinese translation for year.md

### PR DESCRIPTION
**Pull request by kurobanekid**

### CODE REVIEW

File: translations/zh/year.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+2. 你遵守新年的决心了吗？
+3. 你身边有人生孩子了吗？
+4. 你身边有人死了吗？
+5. 你去过哪些城市/州/国家？
+6. 明年你想有什么，而这是你今年缺少的？
+7. 今年的哪些日期将铭刻在你的记忆中，为什么？
+8. 你今年最大的成就是什么？
+9. 你最大的失败是什么？
+10. 你还遇到过哪些困难？
+11. 你生病了还是受伤了？
```
Comment (kaichen) [inline translations/zh/year.md:11]: 我感觉这里表达的是今年是否生过病或者受过伤？用 `X了` 更像是现在时表示当前状态

Comment (kepano) [reply translations/zh/year.md:11]: (reading via Google Translate) the intent of the question was indeed to be in the past tense

**Comment by kepano** [comment]
Reading your translation via Google Translate it seems like you have translated many of the questions as present tense. I'd like to see this be changed to past tense to more closely match the original text. The intent is to reflect on the past 12 months. Can you help make those changes?

File: translations/zh/year.md (hunk 1 above)
Comment (kurobanekid) [reply translations/zh/year.md]: Thank you for your feedback. I have optimized a version.

**Pull request merged**